mlflow/models/evaluation/base.py

```diff
@@ -456,11 +508,121 @@ def _start_run_or_reuse_active_run(run_id):
     else:
         if run_id and active_run.info.run_id != run_id:
             raise ValueError(
-                "An active run exists, you cannot specify another run_id when " "evaluating."
+                "An active run exists, you cannot specify another run_id when evaluating."
             )
         yield active_run.info.run_id
 
 
+def _normalize_evaluators_and_evaluator_config_args(
+        evaluators,
+        evaluator_config,
+):
```

Comment:
Can we write tests for some of the cases here to make sure that `mlflow.evaluate()` behaves as expected? This is important since the logic is complex & previously had some issues.